src/index.js

```diff
@@ -1,17 +1,10 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './app.css';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```

Comment:
- [ ] You should create an app component and add it to the index file. Also, wrap component by React.StrictMode.
- [ ] After creating the app component test it with npm start.

Code will like that.
`ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  document.getElementById('root'),
);`